indico/modules/events/sessions/templates/management/session_list.html

```diff
@@ -0,0 +1,72 @@
+{% extends 'layout/full_content_base.html' %}
+
+{% from 'events/sessions/management/_session_list.html' import render_session_list %}
+
+{% block title %}
+    {%- trans %}Sessions{% endtrans -%}
+{% endblock %}
+
+{% block content %}
+    <div class="sessions">
+        <div class="toolbar">
+            <div class="group left">
+                <a class="icon-checkbox-checked i-button arrow js-dropdown"
+                   href="#" title="{% trans %}Select{% endtrans %}" data-toggle="dropdown"></a>
+                <ul class="dropdown">
+                    <li>
+                        <a href="#" data-select-all="input.select-row">
```

Comment:
I would add to the selector the ID of the element wrapping the table. I find this more explicit: `#session-list input:checkbox`.